typed-builder-macro/src/field_info.rs

```diff
@@ -345,6 +378,58 @@ impl ApplyMeta for SetterSettings {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum StripOption {
+    Span(Span),
+    WithFallback(Span, String),
```

Comment:
Rather than a string, why not use `syn::Ident`? This would also mean the syntax would be
```rust
#[builder(setter(strip_option(fallback = outer_opt))]
```
Which I prefer. I use strings for prefix settings because they get concatenated into other identifiers (which has some implications - e.g. a prefix can be a reserved word) but if the identifier is used as is I prefer to keep it `syn::Ident` all the way.

Reply:
Totally agree, and updated.